pkg/controller/sparkapplication/controller.go

```diff
@@ -102,7 +106,7 @@ func NewController(
 	})
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, apiv1.EventSource{Component: "spark-operator"})
 
-	return newSparkApplicationController(crdClient, kubeClient, crdInformerFactory, podInformerFactory, recorder, metricsConfig, ingressURLFormat, ingressClassName, batchSchedulerMgr, enableUIService)
+	return newSparkApplicationController(crdClient, kubeClient, crdInformerFactory, podInformerFactory, recorder, metricsConfig, ingressURLFormat, ingressClassName, batchSchedulerMgr, enableUIService, enableIngressTLS, ingressTLSAnnotations)
```

Comment:
Ingress related parameters can be grouped together.

Reply:
Oh I probably understand now - by grouping together you mean that if `ingressTLSAnnotations != []` then providin ingress is enabled, and  if `ingressTLSAnnotations == []` it is not?